src/save.js

```diff
@@ -1,14 +1,30 @@
+/**
+ * WordPress dependencies
+ */
+import { useBlockProps } from '@wordpress/block-editor';
 
 const save = ( { attributes } ) => {
 	let cls = '';
-	cls = ( attributes.language ) ? 'language-' + attributes.language : '';
-	cls = ( attributes.lineNumbers ) ? cls + ' line-numbers' : cls;
+	cls = attributes.language ? 'language-' + attributes.language : '';
+	cls = attributes.lineNumbers ? cls + ' line-numbers' : cls;
+
+	const blockProps = useBlockProps?.save( { title: attributes.title } );
 	return (
-		<pre title={ attributes.title }>
-			<code lang={ attributes.language } className={ cls }>
-				{ attributes.content }
-			</code>
-		</pre>
+		<>
+			{ blockProps ? (
+				<pre { ...blockProps }>
+					<code lang={ attributes.language } className={ cls }>
+						{ attributes.content }
+					</code>
+				</pre>
+			) : (
+				<pre className="wp-block-code" title={ attributes.title }>
+					<code lang={ attributes.language } className={ cls }>
+						{ attributes.content }
+					</code>
+				</pre>
+			) }
```

Comment:
There seems to be a migration problem here.

If I create a Code block with Gutenberg 9.2-alpha active:

```
<!-- wp:code -->
<pre class="wp-block-code"><code>if ( true ) {<br>     truth(); <br>}</code></pre>
<!-- /wp:code -->
```

WIthout your plugin active, the editor shows:

![image](https://user-images.githubusercontent.com/134745/96517447-cd246c80-121d-11eb-93e5-65231cf70032.png)

But upon activating your plugin, I get block validation errors:

![image](https://user-images.githubusercontent.com/134745/96517494-df9ea600-121d-11eb-9eec-7155268266da.png)

```
Block validation: Expected text `if ( true ) {<br>     truth(); <br>}`, saw `if ( true ) {`.
```

Again, this seems to be related to the newline problem I reported in https://github.com/WordPress/gutenberg/issues/26301.

Reply:
I don't see that, but I was working off of master not a specific tag. I'll test again shortly, a lot of last minute changes today with the beta cut coming. 😁

Reply:
I was also testing on `master`.

Reply:
I managed to get my fork to be compatible in WP 5.5 and Gutenberg `master`: https://github.com/westonruter/syntax-highlighting-code-block/pull/200.

Reply:
Thanks, this was due to not using the `RichText` component.
I updated reviewing your plugin changes. 👍